pkgs/applications/graphics/akira/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchFromGitHub
+, desktop-file-utils, meson, ninja, pkgconfig, vala_0_40, wrapGAppsHook
+, appstream-glib, cairo, glib, goocanvas2, gtk3, gtksourceview3
+, json-glib, libarchive, libgee, libxml2, pantheon, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "akira";
+  version = "2019-10-12";
+
+  src = fetchFromGitHub {
+    owner = "akiraux";
+    repo = "Akira";
+    rev = "cab952dee4591b6bde34d670c1f853f5a3ff6b19";
+    sha256 = "1fp3a79hkh6xwwqqdrx4zqq2zhsm236c6fhhl5f2nmi108yxz04q";
+  };
+
+  nativeBuildInputs = [ desktop-file-utils meson ninja pkgconfig vala_0_40 wrapGAppsHook ];
```

Comment:
please use the latest vala `vala` attribute.

Reply:
can you also add `vala-lint`  https://github.com/akiraux/Akira/blob/5f4fd22da5c06e1cefaa4859bffeb946874ca117/meson.build#L41 ?

Reply:
Sadly, can't use the latest `vala`: https://github.com/akiraux/Akira/issues/221

Reply:
this patch could fix this https://patch-diff.githubusercontent.com/raw/akiraux/Akira/pull/157.patch

Reply:
there's also https://github.com/akiraux/Akira/commit/bcda8fd53f6f232db0b6411269ba108af551629f.patch, this one is actually committed so maybe it's better. (not sure if it applies though).

Reply:
Sadly, the patch is not enough: https://gist.github.com/filalex77/31b9d5127616fbf710ec27d211626e62

Reply:
> there's also [akiraux/Akira@`bcda8fd`.patch](https://github.com/akiraux/Akira/commit/bcda8fd53f6f232db0b6411269ba108af551629f.patch), this one is actually committed so maybe it's better. (not sure if it applies though).

It applies! Thank you so much :smile: